frontend/test/metabase/lib/expressions/suggest.unit.spec.js

```diff
@@ -283,7 +317,7 @@ describe("metabase/lib/expression/suggest", () => {
         ).toEqual([...AGGREGATION_FUNCTIONS, ...METRICS_ORDERS, OPEN_PAREN]);
       });
 
-      it("should suggest expression operators after aggregation argument", () => {
+      xit("should suggest expression operators after aggregation argument", () => {
```

Comment:
@tlrobinson I'm not sure why this started failing. It looks like it still works in the UI, but this test passes `startRule: "aggregation"` whereas I can only get the UI to pass `"expression"`.

Reply:
Expressions can be used in custom fields (startRule: "expression"), custom aggregations (startRule: "aggregation"), and custom filters (startRule: "boolean").

Add a summarize step then add a custom aggregation.